panel/tests/test_util.py

```diff
@@ -84,8 +86,13 @@ def test_abbreviated_repr_list():
 
 
 def test_abbreviated_repr_ordereddict():
-    assert (abbreviated_repr(OrderedDict([('key', 'some really, really long string')]))
-            == "OrderedDict([('key', ...])")
+    if sys.version_info >= (3, 12):
+        expected = "OrderedDict({'key': 'some ...])"
```

Comment:
I applied this change without thinking... this does not look right 🙃

Reply:
Appears to be passing though.